plugins/window_size/windows/window_size_plugin.cpp

```diff
@@ -64,9 +67,8 @@ EncodableValue GetPlatformChannelRepresentationForMonitor(HMONITOR monitor) {
   MONITORINFO info;
   info.cbSize = sizeof(MONITORINFO);
   GetMonitorInfo(monitor, &info);
-  UINT dpi_x, dpi_y;
-  GetDpiForMonitor(monitor, MDT_EFFECTIVE_DPI, &dpi_x, &dpi_y);
-  double scale_factor = dpi_x / kBaseDpi;
+  INT dpi = FlutterDesktopGetDpiForMonitor(monitor);
+  double scale_factor = static_cast<double>(dpi) / kBaseDpi;
```

Comment:
You shouldn't need the cast here; kBaseDpi is a double in this file. (We could do the same in the other file so we don't need to explicitly cast, but it's not a big deal either way.)